revocation/internal/crl/errors.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crl
+
+import "errors"
+
+var (
+	// ErrDeltaCRLNotChecked is returned when the CRL contains a delta CRL but
+	// the delta CRL are not checked
+	ErrDeltaCRLNotChecked = errors.New("delta CRL are not checked")
```

Comment:
Notation-go may catch this error, then log a warning to indicate that the delta CRL is not checked. Would it be better to name it `ErrDeltaCRLNotChecked`?

Reply:
I guess the confusion comes from the naming itself. Something is `not checked` doesn't necessarily indicate an error. `ErrDeltaCRLNotSupported` sounds like a more proper name for an error?

Reply:
As we discussed, the revocation check should fail when delta CRL is included because it's not supported in this version. If we simply ignore the delta CRL, we may get a final revocation result with status OK, but in fact the delta CRL contains a newly revoked cert. It should be a rare case, but still brings in security concern.

Reply:
Updated to fail the validate when encountered a delta CRL